l1.py

```diff
@@ -398,9 +346,12 @@ def clean(paths: OPPaths):
     Cleans up build outputs, such that trying to deploy the devnet L1 node will proceed as though it
     was the first invocation.
     """
-    if os.path.exists(paths.devnet_l1_gen_dir):
-        print(f"Cleaning up {paths.devnet_l1_gen_dir}")
-        shutil.rmtree(paths.devnet_l1_gen_dir, ignore_errors=True)
+    if os.path.exists(paths.devnet_gen_dir):
+        print(f"Cleaning up {DEVNET_L1_DATA_DIR}")
```

Comment:
Should be `{paths.devnet_gen_dir}` in that string.

Do you think this need to be printed every time we run clean?

Reply:
it is ok for me for print every time

Reply:
The way I'm thinking about this is we should only print things that are useful for the people to know. They probably don't even need to know what these directories are. Ideally they just run the command, the things runs and it's done.

I'm not sure what they can possibly do that is useful when they see "Cleanup up `db/devnet/l1`"

Reply:
actually i am not sure make 'clean' in separate files if is good? should we just make clean logic `roll.py`

Reply:
Agree we should do that!